@xen-orchestra/rest-api/src/groups/group.controller.mts

```diff
@@ -49,4 +56,19 @@ export class GroupController extends XoController<XoGroup> {
   getGroup(@Path() id: string): Promise<Unbrand<XoGroup>> {
     return this.getObject(id as XoGroup['id'])
   }
+
+  /**
+   * @example body { "name": "new group" }
+   */
+  @Example(createGroup)
+  @Post()
+  @SuccessResponse(createdResp.status, createdResp.description)
+  @Response(resourceAlreadyExists.status, resourceAlreadyExists.description)
+  @Response(invalidParameters.status, invalidParameters.description)
+  async createGroup(@Body() groupData: { name: string }): Promise<{ id: string }> {
+    const { name } = groupData
+    const group = await this.restApi.xoApp.createGroup({ name })
```

Comment:
You can pass `groupData` directly.
If the user try to add more parameters that `name: string`, the REST-API throw (normally) an error automatically

```suggestion
    const group = await this.restApi.xoApp.createGroup(groupData)
```